torchao/_models/llama/benchmark_results.txt

```diff
@@ -1,8 +1,16 @@
-20240610164534, tok/s= 94.91, mem/s=1424.58 GB/s, peak_mem=16.43 GB, model_size=15.01 GB quant: None, mod: Meta-Llama-3-8B, compile: True, compile_prefill: False, dtype: torch.bfloat16, device: cuda repro: python generate.py --checkpoint_path ../../../../gpt-fast/checkpoints/meta-llama/Meta-Llama-3-8B/model.pth --device cuda --precision torch.bfloat16 --compile --num_samples 5 --max_new_tokens 200 --top_k 200 --temperature 0.8
```

Comment:
should we keep this file you think? Feels subsumed by the table which is significantly clearer to read

Reply:
having full repros of everything can be nice